documentation/dev/spec/protocol/inventory.md

```diff
@@ -799,6 +799,32 @@ title: Inventory protocol
        <!-- type -->
        <!ELEMENT TYPE (#PCDATA)>>
 
+     <!ELEMENT CAMERAS (RESOLUTIONS, LENSFACING, FLASHUNIT,
+     IMAGEFORMAT, ORIENTATION, FOCALLENGHT, SENSORSIZE, MANUFACTURER,
+     RESOLUTIONVIDEO, MODEL, SUPPORT)>
+       <!-- Camera resolution, ie '1200x800' -->
+       <!ELEMENT RESOLUTIONS (#PCDATA)>
```

Comment:
why a `S` to `RESOLUTIONS` ? Do you mean this can be a list of resolution ? If not, you should better just use `RESOLUTION`.